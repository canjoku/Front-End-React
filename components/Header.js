import { Component } from 'react'
import Navbar from './Navbar'
import SubscriptionForm from './SubscriptionForm'

class Header extends Component {
  render () {
    const { mutation } = this.props
    return (
      <div>
      <Navbar />
      <header>
          <SubscriptionForm mutation={mutation}/>
      </header>
      </div>
    )
  }
}

export default Header
