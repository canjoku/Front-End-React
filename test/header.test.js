import Header from '../components/header'

test('<Header /> renders correctly', () => {
  const component = shallow(<Header />)
  expect(component).toMatchSnapshot()
})
