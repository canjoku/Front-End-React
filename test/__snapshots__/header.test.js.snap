// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<div>
  <ul>
    <li>
      <Link
        href="/index"
      >
        <a>
          Home
        </a>
      </Link>
    </li>
    <li>
      <Link
        href="/article"
      >
        <a>
          Article
        </a>
      </Link>
    </li>
  </ul>
</div>
`;
